src/recladaobject_views.sql

```diff
@@ -0,0 +1,23 @@
+DROP VIEW IF EXISTS reclada.v_object;
+CREATE OR REPLACE VIEW reclada.v_object
+AS
+    SELECT data FROM reclada.object obj
+    WHERE ((data->'revision')::numeric = (SELECT max((objrev.data->'revision')::numeric)
+            FROM reclada.object objrev
+            WHERE (objrev.data->'id' = obj.data->'id')
+                AND (objrev.data->'isDeleted' = 'false')));
+                
+
+DROP VIEW IF EXISTS reclada.v_class;
+CREATE OR REPLACE VIEW reclada.v_class
+AS
+    SELECT data FROM reclada.v_object obj
+    WHERE (data->'class' = '"jsonschema"');
```

Comment:
I wonder if they're any reason to extract value as json object. It involves implicit string to JSON conversion (that's why 2 sets of quotas required) and comparison of JSON objects (which is probably more expensive than simple string comparison)

Reply:
This is because of the index on the class field. We use jsonb for extract class field, so we have the index on this field in jsonb format.

Reply:
As I see it - it should be the other way around.  Indexes was fine, but field extraction operators should be changed to ->> all over the code. If I'm not missing anything.